docs/API.md

```diff
@@ -200,13 +200,14 @@ gulp.task('somename', function() {
 
 #### name
 
-Optional, The name of the task. Tasks that you want to run from the command line
-should not have spaces in them.
-
 If the name is not provided, the task will be named after the function 
-`name` attribute, set on any named function.
+`name` or `displayName` attribute. The name argument is required if `fn` has 
+empty `name` and `displayName` attributes. 
+
+Since the task will be run from the command line, you should avoid using 
```

Comment:
can be run from the command line*

You can also run tasks by name using `gulp.series('a', 'b')` so the docs shouldnt be too focused on the command line